samples/boards/esp32/wifi_apsta_mode/src/main.c

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2023 Muhammad Haziq
```

Comment:
2024?

Reply:
it should be corrected. Let me correct this too.